src/api.tsx

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+const url = 'http://localhost:3003/'
```

Comment:
- let's separate imports and variable declaration
- let's move backend URL into variable in .env file, and use it from .env

Reply:
Done.